linguatec_lexicon/models.py

```diff
@@ -46,28 +46,44 @@ def _clean_search_query(self, query):
 
         return query
 
-    def search(self, query):
+    def search(self, query, lex):
         MIN_SIMILARITY = 0.3
         query = self._clean_search_query(query)
 
+        #Get and use the key of the Lexicon instead of the name
+        if lex != None:
```

Comment:
Avoid using comparators with None. It's more pythonist using expressions like 'is None' or 'is not None'.
Reference https://stackoverflow.com/a/2209781

Reply:
Fixed, thank you for the info